debian/rules

```diff
@@ -31,4 +31,4 @@ override_dh_builddeb:
 override_dh_auto_test:
 	find . -name '*.pyc' -delete
 	find . -name '__pycache__' -delete
-	PATH=.:$$PATH timeout --signal=KILL 60 py.test -vv tests/
+	PATH=.:$$PATH timeout --signal=KILL 60 py.test-3 -vv tests/
```

Comment:
should it be `pytest-3`?

Reply:
I tried that but unfortunately the system-packaged version of pytest is too old on most of the distributions to have that binary (at least xenial doesn't have it, I didn't check bionic).

Reply:
oof, maybe `python3 -m pytest`?

Reply:
(Honestly not sure it's worth even supporting test running during the Debian build since it requires system packages and we get the same test coverage without it. But it's not too big of a hassle...)

Reply:
I just realized we actually only run the build on buster anyway (then run the tests on each platform), so we should be able to change this after all.